lib/tweets.json

```diff
@@ -0,0 +1,245 @@
+[
```

Comment:
If we want we can just use a js array instead of reading a json file, like we do for other static contents